packages/appstore/src/components/wallet-transfer/components/wallet-transfer-messages/wallet-transfer-messages.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useWalletTransfer, useTransferMessageList } from '@deriv/hooks';
+import { localize } from '@deriv/translations';
+import { AnimatedList, AlertMessage } from '@deriv/components';
+import { getAccountName } from 'Constants/utils';
+import { formatMoney } from '@deriv/shared';
+
+type TMessageMapperParams = {
+    currency: string;
+    is_first_transfer: boolean;
+    limit: number;
+    from_name: string;
+    to_name: string;
+};
+
+type TWalletTransferMessagesProps = {
+    from_account: ReturnType<typeof useWalletTransfer>['from_account'];
+    to_account: ReturnType<typeof useWalletTransfer>['to_account'];
+    // setMessageList: void;
+};
+
+const message_code_to_message_mapper: any = {
```

Comment:
`message_code_to_message_mapper` maps the `message_code` given with message data from the `useTransferMessageList` hook.